Gopkg.toml

```diff
@@ -59,7 +59,7 @@
 
 [[constraint]]
   name = "github.com/gogo/protobuf"
-  version = "1.0.0"
+  branch = "master"
```

Comment:
It's generally not a good idea just to use current "master" for all the times you update the dependency. If the changes you want is not available in latest release, could you please pin exact commit here?

Otherwise everything looks good from my side. Maybe @lomik have different opinion on that matter or have a different comments.

Reply:
Master is ok for me